crates/common/tedge_config_macros/impl/src/input/mod.rs

```diff
@@ -0,0 +1,4 @@
+mod parse;
+mod validate;
```

Comment:
Ironically, that's pretty much what validate is doing. It could do with clearer naming to distinguish between "things that can be parsed with syn and darling" and "things that are meaningful for the rest of the application, after e.g. converting boolean flags into enum discriminants" (which is currently in the validate module, even though it's following the idea of "Parse, don't validate").